crates/exex/src/exex.rs

```diff
@@ -381,7 +215,7 @@ mod tests {
 
         // Create a sealed block with a single transaction
         let block = SealedBlockWithSenders {
-            block: SealedBlock { header: header.seal_slow(), body: vec![tx], ..Default::default() },
+            block: SealedBlock { header: header.seal_slow(), body: vec![], ..Default::default() },
```

Comment:
Likewise, I am in favor of keeping a transaction in the block so that we can test everything in real conditions. The ideal would even be to create a follow up issue to have an input JSON with a dozen real blocks in order to always have concrete, simple and easy to turn on unit tests to test that everything passes in Mainnet conditions.

Reply:
agree that I went to fast with the removal of the tx, fixed